crates/editor/src/editor.rs

```diff
@@ -4110,8 +4110,42 @@ impl Editor {
                 active_index: 0,
                 ranges: tabstops,
             });
-        }
 
+            // Check whether the just-entered snippet ends with an auto-closable bracket.
+            if self.autoclose_regions.is_empty() {
+                let snapshot = self.buffer.read(cx).snapshot(cx);
+                for selection in &mut self.selections.all::<Point>(cx) {
+                    let selection_head = selection.head();
+                    let scope = snapshot.language_scope_at(selection_head).unwrap();
```

Comment:
This panics when the tests run.
```suggestion
                    let Some(scope) = snapshot.language_scope_at(selection_head) else {
                        continue;
                    };
```